sklearn/calibration.py

```diff
@@ -388,8 +388,8 @@ def predict_proba(self, X):
         # XXX : for some reason all probas can be 0
         proba[np.isnan(proba)] = 1. / n_classes
 
-        # Deal with cases where the predicted probability minimally exceeds 1.0
-        proba[(1.0 < proba) & (proba <= 1.0 + 1e-5)] = 1.0
+        proba[proba > 1.0] = 1.0
+        proba[proba < 0.0] = 0.0
```

Comment:
I'm not sure why that line was checking for the eps here. Can you check who wrote that and ping them?